checkov/terraform/checks/resource/aws/EKSPlatformVersion.py

```diff
@@ -24,7 +24,7 @@ def get_inspected_key(self) -> str:
         return "version"
 
     def get_expected_values(self) -> list[Any]:
-        return ["1.23", "1.24", "1.25", "1.26", "1.27"]
+        return ["1.24", "1.25", "1.26", "1.27", "1.28"]
```

Comment:
```suggestion
        return ["1.23", "1.24", "1.25", "1.26", "1.27", "1.28"]
```
I'm fine with adding `1.28`, but not about removing `1.23`, because AWS offers extended support for an additional 12 months.

Reply:
Understood!